lib/io.toit

```diff
@@ -734,6 +734,117 @@ class ByteArrayReader_ extends Reader:
   close_ -> none:
     data_ = null
 
+
+interface OutStrategy:
+  /**
+  Writes the given $data to this writer.
+
+  Returns the number of bytes written.
+
+  See $Writer.try-write_.
+  */
+  // This is a protected method. It should not be "private".
+  try-write_ data/Data from/int to/int -> int
+
+  /**
+  Closes this writer.
+
+  See $Writer.close_.
+  */
+  // This is a protected method. It should not be "private".
+  close-writer_ -> none
+
+class Out_ extends Writer:
+  strategy_/OutStrategy
```

Comment:
It isn't very clear to me why you need this extra level of indirection here. Maybe add a comment on the two strategy interfaces to explain why this is convenient?

Reply:
You are right. There is no need for this interface.
Removed it, and I am using `OutMixin` instead.